RIS.js

```diff
@@ -4409,7 +4688,6 @@ var testCases = [
 				"abstractNote": "Optimal integration of next-generation sequencing into mainstream research requires re-evaluation of how problems can be reasonably overcome and what questions can be asked. .... The random sequencing-based approach to identify microsatellites was rapid, cost-effective and identified thousands of useful microsatellite loci in a previously unstudied species.",
 				"issue": "9999",
 				"publicationTitle": "Molecular Ecology Resources",
-				"shortTitle": "Rapid identification of thousands of copperhead snake (Agkistrodon contortrix) microsatellite loci from modest amounts of 454 shotgun genome sequence",
```

Comment:
Hm...

Reply:
Isn't this just that we're not adding a short title when it's the same as the title, only when there's actually a semicolon?

Reply:
Yes, that makes sense.

(I was surprised that this changed compared to the previous run of this test case.)